xtask/src/visualize_bundler_bench.rs

```diff
@@ -64,114 +69,199 @@ pub fn generate(summary_path: PathBuf) -> Result<()> {
+    Dark,
+}
+
+impl Theme {
+    fn name(self) -> &'static str {
+        match self {
+            Theme::Light => "light",
+            Theme::Dark => "dark",
         }
+    }
 
-        chart
-            .configure_mesh()
-            .x_desc("Number of modules")
-            .x_labels(10)
-            .y_labels(10)
-            .y_desc("Mean time (ms) -- lower is better")
-            .draw()?;
-
-        chart
-            .configure_series_labels()
-            .background_style(plotters::style::WHITE)
-            .border_style(plotters::style::BLACK)
-            .position(SeriesLabelPosition::UpperLeft)
-            .margin(10)
-            .draw()?;
-
-        root.present()?;
+    fn legend_background_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::WHITE.into(),
+            Theme::Dark => RGBColor(34, 34, 34).into(),
+        }
     }
 
-    Ok(())
-}
+    fn light_line_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::BLACK.mix(0.1),
+            Theme::Dark => plotters::style::colors::WHITE.mix(0.1),
+        }
+    }
 
-// Avoid Duration to preserve fractional values. Additionally, plotters works
-// directly with f64.
-fn ns_to_ms(ns: f64) -> f64 {
-    ns / 1.0e6
-}
+    fn bold_line_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::BLACK.mix(0.2),
+            Theme::Dark => plotters::style::colors::WHITE.mix(0.2),
+        }
+    }
 
-struct BundlerColor {
-    bundler_to_color: HashMap<String, &'static RGBColor>,
-    iter: Box<dyn Iterator<Item = &'static RGBColor>>,
-}
+    fn axis_line_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::BLACK.into(),
+            Theme::Dark => plotters::style::colors::WHITE.into(),
+        }
+    }
+
+    fn label_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::BLACK.mix(0.75),
+            Theme::Dark => plotters::style::colors::WHITE.mix(0.75),
+        }
+    }
 
-impl BundlerColor {
-    fn new() -> Self {
-        Self {
-            bundler_to_color: HashMap::new(),
-            iter: Box::new(COLORS.iter()),
+    fn axis_desc_color(self) -> RGBAColor {
+        match self {
+            Theme::Light => plotters::style::colors::BLACK.into(),
+            Theme::Dark => plotters::style::colors::WHITE.into(),
         }
     }
+}
+
+const THEMES: [Theme; 2] = [Theme::Light, Theme::Dark];
+
+fn generate_scaling(output_path: PathBuf, by_bench: &ByBench) -> Result<()> {
+    fs::create_dir_all(&output_path)?;
+
+    let colors_by_bundler = HashMap::from([
+        ("Next.js 12 SSR", plotters::style::full_palette::CYAN),
+        ("Next.js 11 SSR", plotters::style::full_palette::BLUE),
+        ("Turbopack SSR", plotters::style::full_palette::RED),
+        ("Vite SWC CSR", plotters::style::full_palette::GREEN),
+    ]);
```

Comment:
This need to have the same keys as `BUNDLER_ORDER`.

Maybe make it an IndexMap and merge both to `BUNDLERS_ORDER_AND_COLOR`.

Also add the other possible bundlers, so it doesn't crash when running the script with them